index.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta charset="utf-8" />
```

Comment:
I'm guessing these changes came from some auto formatting in the editor. I can't see to make out any actual changes to this file, so maybe we want to revert these changes.

Reply:
Yeah, I can checkout the original file. These are visual studio auto-formatting changes.